docs/modules/generic_send.md

```diff
@@ -0,0 +1,77 @@
+# Generic Send (generic_send.py)
+
+## Description
+
+This script is used to send data generated by the [PerunDataGenerator](PerunDataGenerator.md) to the Perun API. It is
```

Comment:
the format of the generated data does not matter, it is not related to the general data generator at all